tests/test_cluster.py

```diff
@@ -2285,5 +2285,22 @@ def test_update(self):
         with self.assertRaises(SystemExit):
             qds.main()
 
+class TestClusterACL(QdsCliTestCase):
+    def test_get_object_policy(self):
+        sys.argv = ['qds.py', 'cluster', 'object_policy', 'get', '--cluster-id', '1001']
+        print_command()
+        Connection._api_call = Mock(return_value={})
+        qds.main()
+        Connection._api_call.assert_called_with('GET', 'object_policy/get_object_policy', params ={'source_id':'1001',
```

Comment:
I dont this will work. `get_object_policy` is not required in the URL. Have you tested it for real ? If it does then the API design is wrong. It should just be `GET object_policy`